code/game/jobs/faction/idris.dm

```diff
@@ -53,7 +53,7 @@
 
 	titles_to_loadout = list(
 		"Security Officer" = /datum/outfit/job/officer/idris,
-		"Forensics Technician" = /datum/outfit/job/forensics/idris,
+		"Forensic Technician" = /datum/outfit/job/forensics/idris,
```

Comment:
Necropolis has this problem too - maybe fix it there as well?

Reply:
Done but they still look like stereotypical bad guys, which I can't exactly change so it's whatever.